grype/db/v6/vulnerability_decorator_store.go

```diff
@@ -14,29 +14,34 @@ import (
 type VulnerabilityDecoratorStoreWriter interface {
 	AddKnownExploitedVulnerabilities(...*KnownExploitedVulnerabilityHandle) error
 	AddEpss(...*EpssHandle) error
+	AddCWE(...*CWEHandle) error
 }
 
 type VulnerabilityDecoratorStoreReader interface {
 	GetKnownExploitedVulnerabilities(cve string) ([]KnownExploitedVulnerabilityHandle, error)
 	GetEpss(cve string) ([]EpssHandle, error)
+	GetCWEs(cve string) ([]CWEHandle, error)
 }
 
 type vulnerabilityDecoratorStore struct {
 	db          *gorm.DB
 	blobStore   *blobStore
 	kevEnabled  bool
 	epssEnabled bool
+	cweEnabled  bool
 	epssDate    *time.Time
 }
 
 func newVulnerabilityDecoratorStore(db *gorm.DB, bs *blobStore, dbVersion schemaver.SchemaVer) *vulnerabilityDecoratorStore {
 	minSupportedKEVClientVersion := schemaver.New(6, 0, 1)
 	minSupportedEPSSClientVersion := schemaver.New(6, 0, 2)
+	minSupportedCWEClientVersion := schemaver.New(6, 0, 0)
```

Comment:
we should put this under test similar to https://github.com/anchore/grype/blob/ff8a98609a3dd67fbab9231a28d428a9c97fdc44/grype/db/v6/vulnerability_decorator_store_test.go#L139